static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -79,7 +79,7 @@ const GroupEventDetailsContent = ({
   const eventEntryProps = {group, event, project};
 
   return (
-    <Fragment>
+    <VisuallyCompleteWithData id="IssueDetails-EventBody" hasData>
```

Comment:
Not passing a condition for `hasData` because this only gets rendered when the event has been loaded and exists.

Reply:
That works 👍 let's give it a shot. You can also move it higher to the larger component if you want to have re-renders w/ loading state etc. captured since `VisuallyCompleteWithData` is also pulling double duty and wrapping the component with a profiler. It should fire at basically the same time since it uses a macro task tick to push it post-render anyway.